types.go

```diff
@@ -224,3 +236,138 @@ type AccountAssociatedAsset struct {
 	Unit     string `json:"unit,omitempty"`
 	Quantity string `json:"quantity,omitempty"`
 }
+
+// MetadataTxLabel return Transaction metadata labels
+// List of all used transaction metadata labels.
+type MetadataTxLabel struct {
+	Label string `json:"label,omitempty"`
+	Cip10 string `json:"cip10,omitempty"`
+	Count string `json:"count,omitempty"`
+}
+
+// MetadataTxContentInJSONRaw Transaction metadata content raw in JSON
+// Transaction metadata per label.
+// This struct are more flexible on JSONMetadata field
+type MetadataTxContentInJSONRaw struct {
+	TxHash string `json:"tx_hash,omitempty"`
+	// 	string or object or Array of any or integer or number or boolean Nullable
+	// Content of the JSON metadata
+	JSONMetadata map[string]interface{} `json:"json_metadata,omitempty"`
```

Comment:
The type for json_metadata can be found here. [swagger.yaml](https://github.com/blockfrost/openapi/blob/master/swagger.yaml)

Looks like this
```
json_metadata:
            anyOf:
              - type: string
              - type: object
              - type: array
                items: {}
              - type: integer
              - type: number
              - type: boolean
            nullable: true
```
Does the current solution support these types? Perhaps include mock tests with those as well.

Reply:
the current soluion only support anyOf objects, but i have an idea of how to solve this

Reply:
Ok